plugin/src/main/java/com/spotify/plugin/dockerfile/AbstractDockerMojo.java

```diff
@@ -401,13 +409,17 @@ private RegistryAuthSupplier createRegistryAuthSupplier() {
     final List<RegistryAuthSupplier> suppliers = new ArrayList<>();
     suppliers.add(new ConfigFileRegistryAuthSupplier());
 
-    try {
-      final RegistryAuthSupplier googleSupplier = googleContainerRegistryAuthSupplier();
-      if (googleSupplier != null) {
-        suppliers.add(0, googleSupplier);
-      }
-    } catch (IOException ex) {
-      getLog().info("ignoring exception while loading Google credentials", ex);
+    if (googleContainerRegistryEnabled){
+       try {
+         final RegistryAuthSupplier googleSupplier = googleContainerRegistryAuthSupplier();
+         if (googleSupplier != null) {
+           suppliers.add(0, googleSupplier);
+         }
+       } catch (IOException ex) {
+         getLog().info("ignoring exception while loading Google credentials", ex);
+       }
+    } else {
+      getLog().info("Google Container Registry support is disabled");
```

Comment:
there is an info line already in googleContainerRegistryAuthSupplier when credentials are successfully loaded, so i wanted to have some parity